dashboard/api.py

```diff
@@ -235,6 +235,7 @@ def get_info_for_course(course, mmtrack):
         "prerequisites": course.prerequisites,
         "has_contact_email": bool(course.contact_email),
         "can_schedule_exam": is_exam_schedulable(mmtrack.user, course),
+        "exam_schedulable_window": get_exam_schedulable_dates(course),
```

Comment:
The way the text is written, I think this we should be displaying the eligible dates instead of the schedulable dates. 

> Are you sure you want to enroll in the exam offered May 12 - May 19, 2021?

Reply:
Eligibility dates are no longer used for anything in the code, and I thought we were going to remove those fields from the model.

Reply:
Well, if we're going to say "Are you sure you want to enroll in the exam offered ... " the expectation is that the dates will be the dates of the exam, not the dates of registering for the exam. 

Maybe we can rewrite that to remove those dates, but I'll have to give it some thought.

Reply:
Do we need to remove the dates? I think we could rewrite the messages to omit the dates, but they seem like a useful reminder for learners.

Reply:
What would the message look like without the dates?

Reply:
I think "this semester" would be pretty understandable.

Reply:
I updated the [Google doc](https://docs.google.com/document/d/1xVwHrYr3WujRATG_76w_WaJ9y51haif0HAAOfBd4evk/edit?ts=605a3d2e#) to reflect that. You can make the changes now, but we should give folks until tomorrow to get back to us.

Reply:
Ok, I got sign-off on removing the dates. I think the copy in the google doc is stable. Let me know if you see any issues.